templates/vars/aks/ithc.json

```diff
@@ -3,22 +3,22 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "kubernetesVersion": {
-      "value": "1.17.9"
+      "value": "1.18.8"
     },
     "nodeMinCount": {
-      "value": 13
+      "value": 10
     },
     "nodeMaxCount": {
-      "value": ""
+      "value": "30"
     },
     "enableAutoScaling": {
-      "value": ""
+      "value": "true"
     },
     "type": {
-      "value": "AvailabilitySet"
+      "value": "VirtualMachineScaleSets"
     },
     "vmSize": {
-      "value": "Standard_D4s_v3"
```

Comment:
We should not change the vmSize in this PR. Autoscaler is not dependent on the vmSzie.

Reply:
reverted it